policy-controller/Cargo.toml

```
@@ -26,3 +26,4 @@ tokio = { version = "1", features = ["rt", "rt-multi-thread", "macros", "parking
 tracing = "0.1"
 tracing-subscriber = { version = "0.2", default-features = false, features = ["env-filter", "fmt", "tracing-log"] }
 warp = { version = "0.3", default-features = false, features = ["tls"] }
+serde = { version = "1", features = ["derive"]}
```

I don't think we actually need this dependency, but if we do, please put it in alphabetical order in this list so that it's next to serde_json :)
```suggestion
serde = { version = "1", features = ["derive"] }
```